src/experiments/tcpsocket/api.js

```diff
@@ -34,7 +34,7 @@ var tcpsocket = class tcpsocket extends ExtensionAPI {
         super(...args);
         ExtensionCommon.defineLazyGetter(this, "TCPSocket", () => {
             const { TCPSocket } = Cu.getGlobalForObject(
-              ChromeUtils.import("resource://gre/modules/Services.jsm")
```

Comment:
Would using just Services work?

Like this: https://searchfox.org/mozilla-central/rev/3424c000a7ff304b2d1efb8561a924232f7f12fc/toolkit/components/aboutprocesses/content/aboutProcesses.js#48

The Services global has been available in this scope since Firefox 88.

Reply:
sounds good :)